robot/src/orchestrator.cpp

```diff
@@ -199,7 +207,12 @@ void robot::Orchestrator::get_new_order()
     tcp::Client order_service_client{options.order_addr, options.order_port};
     order_service_client.send("get_order");
     auto response = order_service_client.receive_blocking();
-    order_rec_time = clock_client->get_current_time();
+    if (response == "no_order") {
```

Comment:
Jeg opdager lige, at der skal et s på. 🙃

```suggestion
    if (response == "no_orders") {
```

Reply:
reeeeee